src/main/java/io/github/jsonSnapshot/SnapshotData.java

```diff
@@ -0,0 +1,32 @@
+package io.github.jsonSnapshot;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeMap;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class SnapshotData {
+  @Getter(AccessLevel.NONE)
+  TreeMap<String, SnapshotDataItem> snapshotDataItems;
```

Comment:
Consider making this private and initializing in the field declaration, that way constructor would not be required.

Reply:
done